makefile/scripts/install_python_requirements

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+#
+# Copyright Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+# Exit script if running from outside of Python Virtual Environment
+[ -z "$VIRTUAL_ENV" ] && echo "This script must be executed from an active Python Virtual Environment, which was not found, exiting..."  && exit 1
+
+# Check if pip CLI is installed in the VENV
+type pip > /dev/null
+pip_found=$?
+if [[ $pip_found != 0 ]]; then
+    echo "pip executable not found in the PATH, exiting..."
+    exit 1
+fi
+
+# Get the full absolute path to the script. Needed while calling the script
+# via various partial paths or sourcing the file from shell
+# BASH_SOURCE[0] is safer then $0 when sourcing.
+# We enter the dirname of the invoked script and then get the current
+# path of the script using pwd
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
can use $BASH_SOURCE instead of the array,  when you are going with the first element in the array (well .. zeroeth :) )

Let me see if I can suggest a simpler-to-read one-liner
```suggestion
SCRIPT_DIR=$(pwd $( realpath $BASH_SOURCE ))
```

Reply:
Will stick with dirname as it's used in other scripts as well, so want to be consistent